scripts/encrypt.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ./node_modules/.bin/babel-node
```

Comment:
Added that to easily encrypt tokens, which helps to setup PayPal. Updated the docs accordingly on https://docs.opencollective.com/help/contributing/development/paypal#developing-testing-sandbox

cc @kewitz